migrations/2022-08-09-142755_create_tables_weather_bot/up.sql

```diff
@@ -2,21 +2,20 @@
 
 -- WEATHER BOT TABLES
 
-CREATE TYPE client_state AS ENUM ('initial', 'set_city', 'find_city' , 'number');
+CREATE TYPE client_state AS ENUM ('initial', 'set_city', 'find_city' , 'find_city_number' , 'set_city_number', 'time', 'offset');
```

Comment:
Just a thought.

maybe it's better to use just strings instead of enum  for client_state.

Because it's not possible to remove values from enum

Reply:
We will continue with enums.